app/src/main/res/layout/activity_request_off.xml

```diff
@@ -136,80 +138,132 @@
                     </FrameLayout>
                 </LinearLayout>
 
-                <LinearLayout
+
+                <FrameLayout
                     android:layout_width="match_parent"
-                    android:layout_height="match_parent"
-                    android:orientation="horizontal"
-                    android:paddingTop="@dimen/dp_7"
-                    android:weightSum="2"
+                    android:layout_height="wrap_content"
                     >
-
-                    <TextView
-                        android:layout_width="match_parent"
-                        android:layout_height="50dp"
-                        android:layout_weight="1"
-                        android:background="@color/colorPrimary"
-                        android:gravity="center"
-                        android:text="@string/off_have_salary"
-                        android:textAllCaps="false"
-                        android:textColor="@android:color/white"
+                    <android.support.design.widget.TextInputLayout
+                        style="@style/StyleTextHint"
+                        >
+                        <EditText
+                            android:hint="Off type"
```

Comment:
WARNING: [HardcodedText] [I18N] Hardcoded string "Off type", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.